Documentation/upgrade.md

```diff
@@ -157,42 +157,54 @@ The agents are deployed and managed by a Kubernetes daemonset. So in order to up
 
 The following command updates the image of the agent daemonset:
 ```bash
-kubectl -n rook-system set image daemonset/rook-agent rook-agent=rook/rook:v0.5.1
+kubectl -n rook-system set image daemonset/rook-agent rook-agent=rook/rook:master
 ```
 
-Once the daemonset is updated, we can begin deleting each agent pod **one at a time** and verifying a new one comes up to replace it that is running the new version.
+Once the command is executed, Kubernetes will begin the flow of updating each of the agent pods.
+
+#### Upgrading from 0.6.x or lower
+If upgrading for a Rook version of 0.6.x or lower, the daemonset pods will need to be manually deleted in order to be replaced with the new version.
+Each agent pod will have to be deleted **one at a time** and verifying a new one comes up to replace it that is running the new version.
```

Comment:
what if we told them to delete the rook-agent daemonset before the upgrade? then the operator will create it again. They are stateless, right?

Reply:
I guess that could work. Let me try this.